.ci/load/Jenkinsfile

```
@@ -0,0 +1,179 @@
+        string(name: "concurrent_requests", defaultValue: "100", description: "The number of concurrent requests to test with")
+        string(name: "duration", defaultValue: "10m", description: "Test duration in minutes. Max: 280 minutes")
+        // num_of_runs currently unsupported
+        // string(name: "num_of_runs", defaultValue: "1", description: "Number of test runs to execute")
+        text(name: "agent_config", "defaultValue": "", description: "Custom APM Agent configuration. (WARNING: May echo to console. Do not supply sensitive data.)")
+        text(name: "locustfile", "defaultValue": "", description: "Locust load-generator plan")
+        booleanParam(name: "local_metrics", description: "Enable local metrics collection?", defaultValue: false)
+        // End script auto-generation
+    }
+
+    stages {
+        stage('Pre-flight'){
+            steps {
+                echo 'Getting authentication information from Vault'
+                withSecretVault(secret: 'secret/apm-team/ci/bandstand', user_var_name: 'APP_TOKEN_TYPE', pass_var_name: 'APP_TOKEN'){
+                    script {
+                        sh(script: ".ci/load/scripts/start.sh", returnStdout: true)
+                        env.LOCUST_RUN_TIME = params.duration
+                        env.LOCUST_USERS = params.concurrent_requests
+                        env.SESSION_TOKEN = sh(script: ".ci/load/scripts/start.sh", returnStdout: true).trim()
+                    }
```

The below will help to get rid of the required script closure 

```suggestion
                    sh(script: ".ci/load/scripts/start.sh", returnStdout: true)
                    setEnvVar('SESSION_TOKEN', sh(script: ".ci/load/scripts/start.sh", returnStdout: true).trim())
```